apm-agent-plugins/apm-spring-resttemplate/apm-spring-resttemplate-test/src/test/java/co.elastic.apm.agent.resttemplate/SpringRestTemplateVersionsIT.java

```diff
@@ -0,0 +1,73 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2020 Elastic and contributors
+ * %%
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ * #L%
+ */
+package co.elastic.apm.agent.resttemplate;
+
+import co.elastic.apm.agent.TestClassWithDependencyRunner;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SpringRestTemplateVersionsIT {
+
+    @ParameterizedTest
+    @CsvSource({
+        "3.0.3.RELEASE", // lower versions are not supported
```

Comment:
Did you manually test 3.0.0 to see that the agent no-ops and does not throw exceptions?

Reply:
This is now checked with integration test, there is no exception thrown nor visible when executing with unsupported versions.